src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_SelectCase.vb

```diff
@@ -238,10 +248,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
 
         ' Rewrite select statement case blocks as IF List
-        Private Function RewriteCaseBlocks(
+        Private Function RewriteCaseBlocksRecursive(
+            selectStatement As BoundSelectStatement,
             generateUnstructuredExceptionHandlingResumeCode As Boolean,
             caseBlocks As ImmutableArray(Of BoundCaseBlock),
-            startFrom As Integer
+            startFrom As Integer,
+            ByRef lazyConditionalBranchLocal As LocalSymbol
```

Comment:
Consider explicitly marking this ```<Out>``` and making sure each code path assigns to this parameter.